### DISCUSSION TITLE
How to get this two new badges?

### DISCUSSION THREAD

**Discussion by MitkoVtori** [community/community#39052] [Other Feature Feedback, Questions, & Ideas] [answered]
### How can I get this two badges and their bronze/silver/gold version?

![image](https://user-images.githubusercontent.com/112943652/201628542-635c3810-d73b-466f-ba28-2ac23b862c52.png)

**Comment by muhammadnurulahsan** [comment, marked as answer]
When you participate in certain programs, GitHub automatically displays a badge on your profile. If you're a registered member of the GitHub Developer Program, building an app with the GitHub API, you'll get a Developer Program Member badge on your profile.

**Comment by EbyJK** [comment]
I saw these two badges appearing on profiles of a few people, but they disappeared the same day itself.

**Comment by Shubham1392003** [comment]
Pick a tag (e.g. python)
Answer questions in that tag
Get upvotes

➡️ System gives:

100 → 🥉
400 → 🥈
1000 → 🥇
